src/client/components/PasswordForm.stories.tsx

```diff
@@ -8,5 +8,11 @@ export default {
   component: PasswordForm,
 } as Meta;
 
-export const Default = () => <PasswordForm submitUrl="" fieldErrors={[]} />;
+export const Default = () => (
+  <PasswordForm
```

Comment:
just thinking having an extra export with an example of a field error might be nice